### PR TITLE
docs: Update roadmap with v1.0 requirements

### PR DESCRIPTION
**Pull request by hspinks**
## What does this PR accomplish?

**Work in progress**

Proposal for review by team leads & more generally. Once we have better consensus, I will make sure each requirement on here is tracked by a GitHub Issue with links from this doc.

## Did you add any dependencies?

N/A

## How did you test the change?

N/A

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -49,45 +49,58 @@ Non-Technical Requirements for v0.1:
...
 * We switched to the Flutter client framework for 0.1, and going forward.
 
-### v0.2
+### v1.0
 
-**Target Date:** March 27, 2020
+**Goal:** Match or exceed the existing functionality in the WHO WhatsApp Bot presented idiomatically with native look-and-feel, with dynamically-updatable content localized to at least one other language besides English and ability to deliver push notifications to users.
 
-**Goal:** v0.2 is not a separate milestone, it simply whatever is in the repo by EOD Friday for preview by the WHO.
+**Target Date:** Monday, March 30, 2020 - app store submission by EOD PDT
```
Comment (advayDev1) [inline docs/ROADMAP.md]:
```suggestion
**Target Date:** Monday, March 30, 2020 - transmission by team to WHO for app store approval by EOD PDT
```

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -49,45 +49,58 @@ Non-Technical Requirements for v0.1:
...
 
-**Target Date:** March 30, 2020
+#### Technical Features / Requirements
 
-**Goal:** In one sentence: v1.0 is a native look-and-feel, dynamically-updatable “port” of the WhatsApp bot content on iOS/Android, with information able to be served specific to your country and language (with at least 2 languages actually implemented), and the ability to push notifications in the future.
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=)
+* Native look-and-feel, designed to present content on mobile screens
+* Localized content translated to at least one other language besides English
+* Offline functionality with default content baked into the app bundle
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
```
Comment (advayDev1) [inline docs/ROADMAP.md]:
```suggestion
* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
* Instrumentation to understand how users are using the app
```

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -49,45 +49,58 @@ Non-Technical Requirements for v0.1:
...
+* Localized push notification capabilities
+  * _(Needs Discussion)_ What are the localization requirements for v1.0? What level of granularity will we need in region (e.g. language, continent, country, city, etc)? How will the user choose their location(s) for push delivery?
+* Analytics capabilities with the ability to collect aggregated data for product improvement complying with all applicable laws & regulations and app policies
+* _(Needs Discussion)_ Distribution pipeline using [fastlane](https://fastlane.tools/) for integration with WHO's existing app distribution process
```
Comment (advayDev1) [inline docs/ROADMAP.md]:
```suggestion
* _(Needs Discussion)_ Distribution pipeline using [fastlane](https://fastlane.tools/) for integration with WHO's existing app distribution process
* Full security review
```

File: docs/ROADMAP.md (hunk 2 above)
Comment (hspinks) [reply docs/ROADMAP.md]: is this not captured in the Analytics bullet? happy to use "instrumentation" terminology if it's preferred, I just wanted to be very specific with what we're capturing

Comment (advayDev1) [reply docs/ROADMAP.md]: No you're right.   I mixed two concepts.  The Q is whether it is a requirement to have *other than* user-behavior analytics (crash reports for example).  I'll remove this comment.

File: docs/ROADMAP.md [hunk 4]
```diff
@@ -49,45 +49,58 @@ Non-Technical Requirements for v0.1:
...
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=)
+* Native look-and-feel, designed to present content on mobile screens
+* Localized content translated to at least one other language besides English
+* Offline functionality with default content baked into the app bundle
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
+* Localized push notification capabilities
+  * _(Needs Discussion)_ What are the localization requirements for v1.0? What level of granularity will we need in region (e.g. language, continent, country, city, etc)? How will the user choose their location(s) for push delivery?
+* Analytics capabilities with the ability to collect aggregated data for product improvement complying with all applicable laws & regulations and app policies
```
Comment (advayDev1) [inline docs/ROADMAP.md]:
```suggestion
* Analytics capabilities with the ability to understand aggregated data about user behavior for product improvement complying with all applicable laws & regulations and app policies
```

The analytics system would still be *collecting* information from each user, but we are simply *using* it in aggregated form.

Comment (hspinks) [reply docs/ROADMAP.md]: @advayDev1 I updated this line to take into account both of your points - hopefully this covers crash reporting too

File: docs/ROADMAP.md [hunk 5]
```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
...
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=)
+* Native look-and-feel, designed to present content on mobile screens
+* Localized content translated to at least one other language besides English
+* Offline functionality with default content baked into the app bundle
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
+* Localized push notification capabilities
+  * _(Needs Discussion)_ What are the localization requirements for v1.0? What level of granularity will we need in region (e.g. language, continent, country, city, etc)? How will the user choose their location(s) for push delivery?
+* Instrumentation capabilities for understanding aggregated useage information to be used for product improvement, complying with all applicable laws & regulations and app policies
```
Comment (epicfaace) [inline docs/ROADMAP.md]:
```suggestion
* Instrumentation capabilities for understanding aggregated usage information to be used for product improvement, complying with all applicable laws & regulations and app policies
```

File: docs/ROADMAP.md [hunk 6]
```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
...
 
-**Target Date:** March 30, 2020
+#### Technical Features / Requirements
 
-**Goal:** In one sentence: v1.0 is a native look-and-feel, dynamically-updatable “port” of the WhatsApp bot content on iOS/Android, with information able to be served specific to your country and language (with at least 2 languages actually implemented), and the ability to push notifications in the future.
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=)
+* Native look-and-feel, designed to present content on mobile screens
+* Localized content translated to at least one other language besides English
+* Offline functionality with default content baked into the app bundle
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
```
Comment (epicfaace) [inline docs/ROADMAP.md]: Is this in v1.0 or vNext? The relevant issue (https://github.com/WorldHealthOrganization/app/issues/24) is currently part of the vNext milestone.

Comment (hspinks) [reply docs/ROADMAP.md]: This is v1.0 - good catch, I've updated the ticket! I think I need to do some general ticket cleanup this evening

File: docs/ROADMAP.md [hunk 7]
```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
   * License Agreement
 * Account information/access from WHO to submit app on behalf of their accounts (also needed for code signing) ([#20](https://github.com/WorldHealthOrganization/app/issues/20) :warning:)
```
Comment (advayDev1) [inline docs/ROADMAP.md]: Remove as this isn't what happened.
```suggestion
```

File: docs/ROADMAP.md [hunk 8]
```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
...
 
-**Goal:** Match or exceed the existing functionality in the WHO WhatsApp Bot presented idiomatically as an app. App content should be localized to the 6 official WHO languages, with the ability to be served from the cloud.
+#### Non-Technical Requirements
 
-Lots of ongoing thoughts & conversations - we will update here when there are more concrete thoughts.
+* Content spec finalized
+* Design spec finalized
+* Legal deliverables
+  * Addendum to privacy policy for analytics
+  * Legal review
```
Comment (advayDev1) [inline docs/ROADMAP.md]:
```suggestion
  * Legal review, including ToS/EULA if required
```

File: docs/ROADMAP.md [hunk 9]
```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
   * License Agreement
```
Comment (advayDev1) [inline docs/ROADMAP.md]:
```suggestion
```

File: docs/ROADMAP.md [hunk 10]
```diff
@@ -13,8 +13,8 @@ There is an urgent, global need for an official WHO App to help contain and miti
 Version numbering terminology:
 
 * **v0.1**: refers to the initial buildout that will be submitted to the app stores but not intended for public release, aiming for submission on Thursday, March 26, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
-* **v0.2**: an informal preview build for WHO by end of day Friday, March 27, 2020.  This is not a separate milestone.
-* **v1.0**: first public release, sent to WHO end of day Monday, March 30, 2020 ([v1.0 milestone](https://github.com/WorldHealthOrganization/app/milestone/4))
+* **v1.0**: public release on the app stores, aiming for submission to app stores on Monday, March 30, 2020 ([v1.0 milestone](https://github.com/WorldHealthOrganization/app/milestone/4))
+* **vNext**: describes features considered for future releases, but not yet prioritized into a specific milestone ([vNext milestone](https://github.com/WorldHealthOrganization/app/milestone/3))
 
 ### v0.1
```
Comment (brunobowden) [inline docs/ROADMAP.md]: v0.1 should be moved to the Release Notes and marked "not publicly released"

You can also use GitHub's release process with notes, for example: https://github.com/google/j2objc/releases - but that's better suited to a library that's pulled in as a dependency. For this, I think a single file with release notes seems more appropriate. Not sure where the file should be and how it should be named. Maybe just move it to the bottom of the ROADMAP?

File: docs/ROADMAP.md [hunk 11]
```diff
@@ -24,23 +24,23 @@ Version numbering terminology:
 
 **GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
```
Comment (brunobowden) [inline docs/ROADMAP.md]: A few lines earlier, add **Release Date:** Thursday, March 27th.... so we met our target!

File: docs/ROADMAP.md (hunk 7 above)
Comment (brunobowden) [reply docs/ROADMAP.md]: Future plan is to setup Fastlane and have that handle app store submission

File: docs/ROADMAP.md [hunk 12]
```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
...
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=)
+* Native look-and-feel, designed to present content on mobile screens
+* Localized content translated to at least one other language besides English
+* Offline functionality with default content baked into the app bundle
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
+* Localized push notification capabilities
+  * _(Needs Discussion)_ What are the localization requirements for v1.0? What level of granularity will we need in region (e.g. language, continent, country, city, etc)? How will the user choose their location(s) for push delivery?
+* Instrumentation capabilities for understanding aggregated usage information to be used for product improvement, complying with all applicable laws & regulations and app policies
+* _(Needs Discussion)_ Distribution pipeline using [fastlane](https://fastlane.tools/) for integration with WHO's existing app distribution process
```
Comment (brunobowden) [inline docs/ROADMAP.md]: Analytics should be included in there as a feature, very difficult to know what we're doing without that

File: docs/ROADMAP.md (hunk 10 above)
Comment (hspinks) [reply docs/ROADMAP.md]: I've put this at the bottom of ROADMAP for now, we can move it out to a separate Release Notes file in the future or use the tags page

File: docs/ROADMAP.md (hunk 12 above)
Comment (hspinks) [reply docs/ROADMAP.md]: I'd used the "Instrumentation" terminology to include analytics, but I added analytics to that bullet point for readability 👍

File: docs/ROADMAP.md [hunk 13]
```diff
@@ -12,82 +12,97 @@ There is an urgent, global need for an official WHO App to help contain and miti
...
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/3
+
+#### Fast Follow-Ons
+
+Features that we will want in upcoming milestones quickly after v1.0 release:
+
+* Triage survey integration
+
+## Release Notes
```
Comment (advayDev1) [inline docs/ROADMAP.md]: I don't think Release Notes should be in a roadmap.  They should be in CHANGELOG.md.

Comment (hspinks) [reply docs/ROADMAP.md]: copy that, thought it'd be easier for this, but I'll pull into a separate file

**Pull request merged**